app/src/main/java/com/github/fribourgsdp/radio/StringComparisons.kt

```diff
@@ -0,0 +1,99 @@
+package com.github.fribourgsdp.radio
```

Comment:
I think the part that compare the string and the part that interpret the comparison should be two different class

Reply:
I had left it to give an insight of what could be done at the very beginning, but it is not useful right now. So we can delete it.

Reply:
Okay Okay